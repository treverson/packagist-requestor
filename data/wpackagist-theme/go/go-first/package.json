{
    "packages": {
        "wpackagist-theme\/go-first": {
            "1.1.0": {
                "name": "wpackagist-theme\/go-first",
                "version": "1.1.0",
                "version_normalized": "1.1.0.0",
                "uid": 153458,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/go-first.1.1.0.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/go-first\/",
                    "reference": "1.1.0"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/go-first\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            },
            "1.2.0": {
                "name": "wpackagist-theme\/go-first",
                "version": "1.2.0",
                "version_normalized": "1.2.0.0",
                "uid": 153459,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/go-first.1.2.0.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/go-first\/",
                    "reference": "1.2.0"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/go-first\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            },
            "1.2.1": {
                "name": "wpackagist-theme\/go-first",
                "version": "1.2.1",
                "version_normalized": "1.2.1.0",
                "uid": 153460,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/go-first.1.2.1.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/go-first\/",
                    "reference": "1.2.1"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/go-first\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            },
            "1.2.2": {
                "name": "wpackagist-theme\/go-first",
                "version": "1.2.2",
                "version_normalized": "1.2.2.0",
                "uid": 153461,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/go-first.1.2.2.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/go-first\/",
                    "reference": "1.2.2"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/go-first\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            },
            "1.2.3": {
                "name": "wpackagist-theme\/go-first",
                "version": "1.2.3",
                "version_normalized": "1.2.3.0",
                "uid": 153462,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/go-first.1.2.3.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/go-first\/",
                    "reference": "1.2.3"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/go-first\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            },
            "1.2.4": {
                "name": "wpackagist-theme\/go-first",
                "version": "1.2.4",
                "version_normalized": "1.2.4.0",
                "uid": 153463,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/go-first.1.2.4.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/go-first\/",
                    "reference": "1.2.4"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/go-first\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            },
            "1.2.5": {
                "name": "wpackagist-theme\/go-first",
                "version": "1.2.5",
                "version_normalized": "1.2.5.0",
                "uid": 153464,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/go-first.1.2.5.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/go-first\/",
                    "reference": "1.2.5"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/go-first\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            },
            "1.2.6": {
                "name": "wpackagist-theme\/go-first",
                "version": "1.2.6",
                "version_normalized": "1.2.6.0",
                "uid": 153465,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/go-first.1.2.6.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/go-first\/",
                    "reference": "1.2.6"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/go-first\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            },
            "1.2.7": {
                "name": "wpackagist-theme\/go-first",
                "version": "1.2.7",
                "version_normalized": "1.2.7.0",
                "uid": 153466,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/go-first.1.2.7.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/go-first\/",
                    "reference": "1.2.7"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/go-first\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            },
            "1.2.10": {
                "name": "wpackagist-theme\/go-first",
                "version": "1.2.10",
                "version_normalized": "1.2.10.0",
                "uid": 153467,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/go-first.1.2.10.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/go-first\/",
                    "reference": "1.2.10"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/go-first\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            },
            "1.2.11": {
                "name": "wpackagist-theme\/go-first",
                "version": "1.2.11",
                "version_normalized": "1.2.11.0",
                "uid": 153468,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/go-first.1.2.11.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/go-first\/",
                    "reference": "1.2.11"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/go-first\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            },
            "1.3.1": {
                "name": "wpackagist-theme\/go-first",
                "version": "1.3.1",
                "version_normalized": "1.3.1.0",
                "uid": 153469,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/go-first.1.3.1.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/go-first\/",
                    "reference": "1.3.1"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/go-first\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            },
            "1.4.0": {
                "name": "wpackagist-theme\/go-first",
                "version": "1.4.0",
                "version_normalized": "1.4.0.0",
                "uid": 153470,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/go-first.1.4.0.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/go-first\/",
                    "reference": "1.4.0"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/go-first\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            }
        }
    }
}