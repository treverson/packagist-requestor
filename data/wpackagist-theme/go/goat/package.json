{
    "packages": {
        "wpackagist-theme\/goat": {
            "0.1": {
                "name": "wpackagist-theme\/goat",
                "version": "0.1",
                "version_normalized": "0.1.0.0",
                "uid": 153618,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/goat.0.1.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/goat\/",
                    "reference": "0.1"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/goat\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            }
        }
    }
}