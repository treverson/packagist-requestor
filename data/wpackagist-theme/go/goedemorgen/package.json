{
    "packages": {
        "wpackagist-theme\/goedemorgen": {
            "0.9.0": {
                "name": "wpackagist-theme\/goedemorgen",
                "version": "0.9.0",
                "version_normalized": "0.9.0.0",
                "uid": 153672,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/goedemorgen.0.9.0.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/goedemorgen\/",
                    "reference": "0.9.0"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/goedemorgen\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            },
            "0.9.1": {
                "name": "wpackagist-theme\/goedemorgen",
                "version": "0.9.1",
                "version_normalized": "0.9.1.0",
                "uid": 153673,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/goedemorgen.0.9.1.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/goedemorgen\/",
                    "reference": "0.9.1"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/goedemorgen\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            },
            "0.9.2": {
                "name": "wpackagist-theme\/goedemorgen",
                "version": "0.9.2",
                "version_normalized": "0.9.2.0",
                "uid": 153674,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/goedemorgen.0.9.2.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/goedemorgen\/",
                    "reference": "0.9.2"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/goedemorgen\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            },
            "0.9.3": {
                "name": "wpackagist-theme\/goedemorgen",
                "version": "0.9.3",
                "version_normalized": "0.9.3.0",
                "uid": 153675,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/goedemorgen.0.9.3.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/goedemorgen\/",
                    "reference": "0.9.3"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/goedemorgen\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            },
            "0.9.4": {
                "name": "wpackagist-theme\/goedemorgen",
                "version": "0.9.4",
                "version_normalized": "0.9.4.0",
                "uid": 153676,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/goedemorgen.0.9.4.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/goedemorgen\/",
                    "reference": "0.9.4"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/goedemorgen\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            },
            "0.9.5": {
                "name": "wpackagist-theme\/goedemorgen",
                "version": "0.9.5",
                "version_normalized": "0.9.5.0",
                "uid": 153677,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/goedemorgen.0.9.5.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/goedemorgen\/",
                    "reference": "0.9.5"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/goedemorgen\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            },
            "0.9.6": {
                "name": "wpackagist-theme\/goedemorgen",
                "version": "0.9.6",
                "version_normalized": "0.9.6.0",
                "uid": 153678,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/goedemorgen.0.9.6.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/goedemorgen\/",
                    "reference": "0.9.6"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/goedemorgen\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            },
            "0.9.7": {
                "name": "wpackagist-theme\/goedemorgen",
                "version": "0.9.7",
                "version_normalized": "0.9.7.0",
                "uid": 153679,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/goedemorgen.0.9.7.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/goedemorgen\/",
                    "reference": "0.9.7"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/goedemorgen\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            },
            "0.9.8": {
                "name": "wpackagist-theme\/goedemorgen",
                "version": "0.9.8",
                "version_normalized": "0.9.8.0",
                "uid": 153680,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/goedemorgen.0.9.8.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/goedemorgen\/",
                    "reference": "0.9.8"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/goedemorgen\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            },
            "0.9.9": {
                "name": "wpackagist-theme\/goedemorgen",
                "version": "0.9.9",
                "version_normalized": "0.9.9.0",
                "uid": 153681,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/goedemorgen.0.9.9.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/goedemorgen\/",
                    "reference": "0.9.9"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/goedemorgen\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            },
            "1.0.0": {
                "name": "wpackagist-theme\/goedemorgen",
                "version": "1.0.0",
                "version_normalized": "1.0.0.0",
                "uid": 153682,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/goedemorgen.1.0.0.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/goedemorgen\/",
                    "reference": "1.0.0"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/goedemorgen\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            },
            "1.0.1": {
                "name": "wpackagist-theme\/goedemorgen",
                "version": "1.0.1",
                "version_normalized": "1.0.1.0",
                "uid": 153683,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/goedemorgen.1.0.1.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/goedemorgen\/",
                    "reference": "1.0.1"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/goedemorgen\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            },
            "1.0.2": {
                "name": "wpackagist-theme\/goedemorgen",
                "version": "1.0.2",
                "version_normalized": "1.0.2.0",
                "uid": 153684,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/goedemorgen.1.0.2.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/goedemorgen\/",
                    "reference": "1.0.2"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/goedemorgen\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            },
            "1.0.3": {
                "name": "wpackagist-theme\/goedemorgen",
                "version": "1.0.3",
                "version_normalized": "1.0.3.0",
                "uid": 153685,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/goedemorgen.1.0.3.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/goedemorgen\/",
                    "reference": "1.0.3"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/goedemorgen\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            },
            "1.0.4": {
                "name": "wpackagist-theme\/goedemorgen",
                "version": "1.0.4",
                "version_normalized": "1.0.4.0",
                "uid": 153686,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/goedemorgen.1.0.4.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/goedemorgen\/",
                    "reference": "1.0.4"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/goedemorgen\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            },
            "1.0.5": {
                "name": "wpackagist-theme\/goedemorgen",
                "version": "1.0.5",
                "version_normalized": "1.0.5.0",
                "uid": 153687,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/goedemorgen.1.0.5.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/goedemorgen\/",
                    "reference": "1.0.5"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/goedemorgen\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            },
            "1.0.6": {
                "name": "wpackagist-theme\/goedemorgen",
                "version": "1.0.6",
                "version_normalized": "1.0.6.0",
                "uid": 153688,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/goedemorgen.1.0.6.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/goedemorgen\/",
                    "reference": "1.0.6"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/goedemorgen\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            },
            "1.0.7": {
                "name": "wpackagist-theme\/goedemorgen",
                "version": "1.0.7",
                "version_normalized": "1.0.7.0",
                "uid": 153689,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/goedemorgen.1.0.7.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/goedemorgen\/",
                    "reference": "1.0.7"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/goedemorgen\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            },
            "1.0.8": {
                "name": "wpackagist-theme\/goedemorgen",
                "version": "1.0.8",
                "version_normalized": "1.0.8.0",
                "uid": 153690,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/goedemorgen.1.0.8.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/goedemorgen\/",
                    "reference": "1.0.8"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/goedemorgen\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            }
        }
    }
}