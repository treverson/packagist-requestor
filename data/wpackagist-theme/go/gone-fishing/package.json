{"packages":{"wpackagist-theme\/gone-fishing":{"2.0":{"name":"wpackagist-theme\/gone-fishing","version":"2.0","version_normalized":"2.0.0.0","uid":153972,"dist":{"type":"zip","url":"https:\/\/downloads.wordpress.org\/theme\/gone-fishing.2.0.zip"},"source":{"type":"svn","url":"https:\/\/themes.svn.wordpress.org\/gone-fishing\/","reference":"2.0"},"homepage":"https:\/\/wordpress.org\/themes\/gone-fishing\/","require":{"composer\/installers":"~1.0"},"type":"wordpress-theme"}}}}