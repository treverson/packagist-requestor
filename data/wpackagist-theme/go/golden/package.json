{
    "packages": {
        "wpackagist-theme\/golden": {
            "1.0.0": {
                "name": "wpackagist-theme\/golden",
                "version": "1.0.0",
                "version_normalized": "1.0.0.0",
                "uid": 153783,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/golden.1.0.0.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/golden\/",
                    "reference": "1.0.0"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/golden\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            },
            "1.3.1": {
                "name": "wpackagist-theme\/golden",
                "version": "1.3.1",
                "version_normalized": "1.3.1.0",
                "uid": 153784,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/golden.1.3.1.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/golden\/",
                    "reference": "1.3.1"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/golden\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            },
            "1.3.2": {
                "name": "wpackagist-theme\/golden",
                "version": "1.3.2",
                "version_normalized": "1.3.2.0",
                "uid": 153785,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/golden.1.3.2.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/golden\/",
                    "reference": "1.3.2"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/golden\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            },
            "1.3.3": {
                "name": "wpackagist-theme\/golden",
                "version": "1.3.3",
                "version_normalized": "1.3.3.0",
                "uid": 153786,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/golden.1.3.3.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/golden\/",
                    "reference": "1.3.3"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/golden\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            },
            "1.3.4": {
                "name": "wpackagist-theme\/golden",
                "version": "1.3.4",
                "version_normalized": "1.3.4.0",
                "uid": 153787,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/golden.1.3.4.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/golden\/",
                    "reference": "1.3.4"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/golden\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            },
            "1.4.1": {
                "name": "wpackagist-theme\/golden",
                "version": "1.4.1",
                "version_normalized": "1.4.1.0",
                "uid": 153788,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/golden.1.4.1.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/golden\/",
                    "reference": "1.4.1"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/golden\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            },
            "1.6.13": {
                "name": "wpackagist-theme\/golden",
                "version": "1.6.13",
                "version_normalized": "1.6.13.0",
                "uid": 153789,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/golden.1.6.13.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/golden\/",
                    "reference": "1.6.13"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/golden\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            },
            "1.8.11": {
                "name": "wpackagist-theme\/golden",
                "version": "1.8.11",
                "version_normalized": "1.8.11.0",
                "uid": 153790,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/golden.1.8.11.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/golden\/",
                    "reference": "1.8.11"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/golden\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            },
            "1.8.13": {
                "name": "wpackagist-theme\/golden",
                "version": "1.8.13",
                "version_normalized": "1.8.13.0",
                "uid": 153791,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/golden.1.8.13.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/golden\/",
                    "reference": "1.8.13"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/golden\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            },
            "1.11.6": {
                "name": "wpackagist-theme\/golden",
                "version": "1.11.6",
                "version_normalized": "1.11.6.0",
                "uid": 153792,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/golden.1.11.6.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/golden\/",
                    "reference": "1.11.6"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/golden\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            },
            "2.1.5": {
                "name": "wpackagist-theme\/golden",
                "version": "2.1.5",
                "version_normalized": "2.1.5.0",
                "uid": 153793,
                "dist": {
                    "type": "zip",
                    "url": "https:\/\/downloads.wordpress.org\/theme\/golden.2.1.5.zip"
                },
                "source": {
                    "type": "svn",
                    "url": "https:\/\/themes.svn.wordpress.org\/golden\/",
                    "reference": "2.1.5"
                },
                "homepage": "https:\/\/wordpress.org\/themes\/golden\/",
                "require": {
                    "composer\/installers": "~1.0"
                },
                "type": "wordpress-theme"
            }
        }
    }
}